src/Rules/Library/BoundingRectangleSizeReasonable.cs

```diff
@@ -26,6 +26,10 @@ public BoundingRectangleSizeReasonable()
 
         public override bool PassesTest(IA11yElement e)
         {
+            if (e?.ItemStatus?.Contains("<Property Name=\"DataContext\" Value=\"Telerik.Windows.Controls.Sparklines.SparklineColumnDataPoint\" />") == true)
```

Comment:
@chingucoding, is there a reason not to put this in the Condition as we called out as the plan in the issue?

Reply:
Nope not at all, I somehow forgot that.